lib/features/authentication/view/pages/onboarding/onboarding_pages.dart

```diff
@@ -14,14 +14,17 @@ class OnboardingPages extends ConsumerWidget {
   static const routePath = '/onboardingpage';
 
   const OnboardingPages({super.key});
+
   @override
   Widget build(BuildContext context, WidgetRef ref) {
     final constants1 = ref.watch(onboarding1Provider);
     final constants2 = ref.watch(onboarding2Provider);
     final constants3 = ref.watch(onboarding3Provider);
     final image = ref.watch(imageConstantsProvider);
+    final pageController = PageController(initialPage: 0);
```

Comment:
Use hooks for page controller or use stateful widget and dispose the controller properly